framework/db/mysql/Schema.php

```diff
@@ -174,7 +174,7 @@ protected function loadColumnSchema($info)
         $column->phpType = $this->getColumnPhpType($column);
 
         if (!$column->isPrimaryKey) {
-            if ($column->type === 'timestamp' && $info['default'] === 'CURRENT_TIMESTAMP') {
+            if (($column->type === 'timestamp' || $column->type ==='datetime') && $info['default'] === 'CURRENT_TIMESTAMP') {
```

Comment:
is this valid in MySQL? afair CURRENT_TIMESTAMP only works with timestamp columns. My knowledge here could be outdated but we should verify that.